macos/FluentUI/Button/Button.swift

```diff
@@ -154,6 +154,39 @@ open class Button: NSButton {
 		}
 	}
 
+	/// When set to a non `nil` value ,the Secondary Image will always be placed on the trailing edge of the button in both LTR and
+	/// RTL language settings. It is designed to remain independent of  the `imagePosition` value of the NSButton which deals
+	/// with the main Button `image`.
```

Comment:
Minor: Instead of "deals with the main Button 'image', you could say "applies to the built-in 'image' property for NSButtons." Or just "applies to the built-in 'image' property."